spec/presenters/instance_presenter_spec.rb

```diff
@@ -32,6 +26,40 @@
 
     expect(instance_presenter.site_contact_email).to eq "admin@example.com"
   end
+  
+  context "when Setting.open_registrations is false" do
+    it "closes registrations" do
+      Setting.open_registrations = false
```

Comment:
If you change `Setting...` values in specs, you should make sure to restore them to their original values to not have side effects elsewhere.